docs/features/job-queue.txt

```diff
@@ -1,103 +1,159 @@
+   pool of other tasks, cancel the task and resubmit it.
+
+-  Changing priority in Kubernetes always implicitly cancels the job, including when the job
+   priority is increased. Priority change in Kubernetes is limited to experiments.
+
+-  Changing resource pools is not available on Kubernetes because it does not currently support
+   multiple resource pools.
 
 ************
  Job States
 ************
 
 Queued jobs can be in the ``Queued`` or ``Scheduled`` state:
 
--  ``Queued``: Job received but resources not allocated
--  ``Scheduled``: Scheduled to run or running, and resources may have been allocated.
++---------------+-----------------------------------------------------------------------+
+| State         | Description                                                           |
++===============+=======================================================================+
+| ``Queued``    | Job received but resources not allocated.                             |
++---------------+-----------------------------------------------------------------------+
+| ``Scheduled`` | Scheduled to run or running, and resources might have been allocated. |
++---------------+-----------------------------------------------------------------------+
 
-Completed or errored jobs are not counted as active and are omitted from this list.
+Completed or errored jobs are not counted as active and are omitted from the queued job list.
 
-*************************
- Visualize the Job Queue
-*************************
+********************
+ View the Job Queue
+********************
 
 You can view the job queue using the CLI or WebUI. In the WebUI, click the **Job Queue** tab. In the
 CLI, use one of the following commands:
 
-.. code::
+.. code:: bash
 
    $ det job list
+
+or
+
+.. code:: bash
+
    $ det job ls
 
 These commands show the default resource pool queue. To view other resource pool queues, use the
 ``--resource-pool`` option, specifying the pool:
 
-.. code::
+.. code:: bash
 
    $ det job list --resource-pool compute-pool
 
 For more information about the CLI options, see the CLI documentation or use the ``det job list -h``
 command.
 
-The WebUI and the CLI display a table of results, ordered by scheduling order. The scheduling order
-does not represent the job priority. In addition to job order, the table includes the job states and
-number of slots allocated to each job.
+The WebUI and the CLI display a results table ordered by scheduling order. The scheduling order does
+not represent the job priority. In addition to job order, the listing shows:
+
++---------------+------------------------------------------------------------------------------+
+| Job Property  | Description                                                                  |
++===============+==============================================================================+
+| ``ID``        | TBD                                                                          |
```

Comment:
The unique ID assigned to the Job.

Reply:
Not super sure if these should be full sentences or as succinct as possible, but hopefully these are ok.